packages/components/src/avatar/avatar.tsx

```diff
@@ -34,7 +38,16 @@ const Avatar = (props: Props) => {
   }, [src])
 
   return (
-    <Base size={size} shape={shape} outline={outline}>
+    <Base
+      size={size}
+      shape={shape}
+      style={{
+        background: identiconRing,
```

Comment:
does not exists?

`Object literal may only specify known properties, and 'background' does not exist in type 'ViewStyle | RecursiveArray<ViewStyle | Falsy | RegisteredStyle<ViewStyle>>'.ts(2322)`